lib/chef/knife/bootstrap.rb

```diff
@@ -836,6 +838,7 @@ def ssh_opts
         return opts if connection_protocol == "winrm"
         opts[:non_interactive] = true # Prevent password prompts from underlying net/ssh
         opts[:forward_agent] = (config_value(:ssh_forward_agent) === true)
+        opts[:connection_timeout] = config_value(:session_timeout)&.to_i || 60
```

Comment:
Any reason not to just make the default 60 seconds and avoid this assignment here?

Reply:
Hey @tas50 As the default `connection_timeout` value is 15 secs (https://github.com/inspec/train/blob/827b73a66ae10b7c54de9fef9a228862f425bb43/lib/train/transports/ssh.rb#L55) 

And the only reason to assign a default value to 60 secs due to previously we passed for `--winrm-session-timeout` 60 secs.

Any suggestion would be helpful?

Thanks

Reply:
yeah we should just set it to 60 for knife, but we can do that via CLI default unless I'm missing something

Reply:
@tas50 Will check and update accordingly.